src/test/java/com/google/sps/data/DatastoreClientTest.java

```diff
@@ -44,21 +40,116 @@ public void tearDown() {
   }
 
   @Test
-  public void testInstance() {
+  public void testAttendeeIsInsertedToDatastore() {
+    // Tests that object is inserted to datastore.
+    DatastoreClientInterface datastoreClient = new DatastoreClient();
+    AttendeeInterface attendee = new Attendee("12345", "Taniece", new Date());
+    datastoreClient.insertOrUpdateAttendee(attendee);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Assert.assertEquals(1, datastore.prepare(new Query
+        (EntityConstants.AttendeeEntity.TABLE_NAME))
+        .countEntities(withLimit(10)));
+    // Tests that getAttendee finds and returns attendee.
+    AttendeeInterface attendee2 =
+        datastoreClient.getAttendee("Taniece").get();
+    Assert.assertEquals(attendee, attendee2);
+    // Tests that existing attendees are updated after being modified.
+    datastoreClient.insertOrUpdateAttendee(attendee2);
+    Assert.assertEquals(1, datastore.prepare(new Query
+        (EntityConstants.AttendeeEntity.TABLE_NAME))
+        .countEntities(withLimit(10)));
+  }
+
+  @Test
+  public void testSessionIsInsertedToDatastore() {
+    // Tests that object is inserted to datastore.
+    DatastoreClientInterface datastoreClient = new DatastoreClient();
+    SessionInterface session = new Session("12345", 
+        Optional.of("Taniece"), Optional.of("12345"));
+    datastoreClient.insertOrUpdateSession(session);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Assert.assertEquals(1, datastore.prepare(new Query
+        (EntityConstants.SessionEntity.TABLE_NAME))
+        .countEntities(withLimit(10)));
+    // Tests that getSession finds and returns session.
+   SessionInterface session2 = null;
+      session2 = datastoreClient.getSession("12345").get();   
+    Assert.assertEquals(session, session2);
+    // Tests that existing sessions are updated.
+    session2.setIpOfVM("54321");
+    datastoreClient.insertOrUpdateSession(session2);
+    Assert.assertEquals(1, datastore.prepare(new Query
+        (EntityConstants.SessionEntity.TABLE_NAME))
+        .countEntities(withLimit(10)));
```

Comment:
No Yoda conditions. 

https://en.wikipedia.org/wiki/Yoda_conditions#:~:text=In%20programming%20jargon%2C%20Yoda%20conditions,side%20of%20the%20conditional%20statement.

Instead of assertEquals(1, blah), do: assertEquals(blah, 1)

Reply:
Change that throughout the PR